### PR TITLE
Liam's 20.04 updates

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

- Fixed up Liam's branch

## QA

1. download this repo
2. test the updated pages

@zprood can you see if this is closer to what you want?

### CODE REVIEW

**Comment by zprood** [comment]
@pmahnke Yes, thanks a lot. Can we have demo for this ?

**Comment by pmahnke** [comment]
@zprood https://cn-ubuntu-com-canonical-web-and-design-pr-404.run.demo.haus/

**Comment by zprood** [comment]
@pmahnke Great, all good! I'm sorry that there's a wrong word while translating. Would you help to update it ? 
**Directory: templates/server/index.html** : 
From:
<li class="p-list__item is-ticked">更为灵魂的的bootloader，可容许故障磁盘启动。</li>
To : 
<li class="p-list__item is-ticked">更为灵活的bootloader，可容许故障磁盘启动。</li>

Then, we can live it. 👍

File: templates/download/thank-you/index.html [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+     </div>
+    </section>
+
+  <div class="p-strip">
+      <div class="row">
+        <h3>文档和资源</h3>
+      </div>
+        <div class="row u-equal-height">
+          <div class="col-4">
+            <h4><a href="http://www.ubuntu.com/server/docs">Ubuntu Server指南&nbsp;›</a></h4>
```
Comment (anthonydillon) [inline templates/download/thank-you/index.html:162]: This one should be external

File: templates/server/index.html [hunk 2]
```diff
@@ -1,460 +1,512 @@
...
-      </div>
+</section>
+
+<div class="p-strip is-bordered">
+    <div class="u-fixed-width">
+      <h2>20.04 LTS新功能</h2>
+  <ul class="p-list is-split">
+    <li class="p-list__item is-ticked">由Canonical提供5年的支持至2025年，ESM扩展支持至2030年。</li>
+    <li class="p-list__item is-ticked">可在主流平台上运行——x86-64，ARM v7，ARM64，POWER8，POWER9，IBM s390x（LinuxONE），且将引入对RISC-V的基础支持。</li>
+    <li class="p-list__item is-ticked">为<a href="/aws">AWS</a> and <a href="/azure">Azure</a>所定制的云镜像——Ubuntu Pro，包含了安全加固，认证，内核热补丁(Kernel livepatch)以及其他功能。</li>
```
Comment (anthonydillon) [inline templates/server/index.html:35]: These links 404

File: templates/download/thank-you/index.html (hunk 1 above)
Comment (zprood) [reply templates/download/thank-you/index.html]: Updated

File: templates/server/index.html (hunk 2 above)
Comment (zprood) [reply templates/server/index.html]: Updated

**Pull request merged**